usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -29,6 +29,7 @@ The currently available endpoints are listed in the following table.
 | Endpoint | Methods | Description |
 | -------- | ------- | ----------- |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/](/api/v2/agency/012/)|GET| Returns agency overview information for USAspending.gov's Agency Details page for agencies that have ever awarded. |
+|[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/object_class/count/](/api/v2/agency/012/object_class/count/)|GET| Returns the count of unique Object Classes for the agency in a single fiscal year |
```

Comment:
Simlar to https://github.com/fedspendingtransparency/usaspending-api/pull/2449#discussion_r418642354, will this make sense to a user?

Reply:
I admit I should have seen it in the Program Activity PR this was basing off of.